src/Stratis.Features.FederatedPeg/Conversion/ConversionRequest.cs

```diff
@@ -58,6 +67,9 @@ public class ConversionRequest : IBitcoinSerializable
         /// </summary>
         public string DestinationAddress { get { return this.destinationAddress; } set { this.destinationAddress = value; } }
 
+        /// <summary>Chain on which STRAX minting or burning should occur.</summary>
+        public DestinationChain DestinationChain { get { return (DestinationChain)this.destinationChain; } set { this.destinationChain = (int)value; } }
```

Comment:
If we are introducing this new field we will ideally need a migration mechanism for existing conversion requests stored by the multisig nodes.

Reply:
Given that we now have almost no conversion requests and that we can have multisig nodes upgraded in a few hours I think we can do without it at this stage. 

But I agree, migration mechanism will be needed later on